<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>60</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>58</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>60</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowOpacity">
   <double>0.500000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="checkBox_layerEnable">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Default</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_layerRange">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>0-10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="layerPrioritySlider">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_2">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_layerPool">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Whole System</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Redshift</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>This Machine</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_layerPacketSize">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>layerPrioritySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>layerPrioritySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_layerEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_layerRange</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>19</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_layerEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>line_4</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_layerEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>line_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_layerEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>layerPrioritySlider</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_layerEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_layerEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>line_3</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>19</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>721</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_layerEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_layerPacketSize</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>753</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_layerEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox_layerPool</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>690</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_layerEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>line_3</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>721</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_layerEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>line_4</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_layerEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>line_5</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>607</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
